style/responsiveness.css

```diff
@@ -19,4 +15,93 @@
         flex-direction: column;
         gap: 3;
     }
+    .header__item {
+        padding: 2rem 23px;
+    }
+    .header__navigation {
+        display: none;
+    }
+
+    .section-header {
+        text-align: center;
+        font-size: 40px;
+        margin-top: 2rem;
+    }
+    .header__button-container {
+        flex-direction: column;
+    }
+    .section-header {
+        font-size: 40px;
+        margin: 3rem 0rem ;
+    }
+  
+    /* ======= Start: containers ====== */
+    .contact-us{
+        background-size: cover;
+    }
+    .container {
+        margin-top: 1rem;
+        padding: 0px;
+    }
+    .container--spacing-larg {
+        margin-top:3rem;
+    }
+    .header__content {
```

Comment:
restructure this CSS file bro. where `contact-us` is styled , is not where you can find the header's elements. improve this as well